ui/src/core_components/CorePDF.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+	<div class="CorePDF">
+		<object class="pdf" :data="fields.source.value" type="application/pdf" width="100%" height="100%">
+			<p>
+				You don't have a PDF plugin, but you can
+				<a :href="fields.source.value">download the PDF file. </a>
+			</p>
+		</object>
+		<div class="mask" />
+	</div>
+</template>
+
+<script lang='ts'>
+import { FieldType } from "../streamsyncTypes";
+import { cssClasses } from "../renderer/sharedStyleFields";
+
+const description =
+	"A component to embed a PDF document.";
+
+export default {
+	streamsync: {
+		name: "PDF",
+		description,
+		category: "Embed",
+		fields: {
+			source: {
+				name: "Pdf source",
```

Comment:
Sorry for being a bit too intense with this stuff but can you capitalise PDF here?